back-office/customer-service/customer-service/dashboard.md

```diff
@@ -0,0 +1,370 @@
+
+There is 5 options, see the improvement [#19197](https://github.com/PrestaShop/PrestaShop/issues/19197)
+
+1) Mark as "opened"
+
+After clicking on it, the status of the selected messages are changed for "opened" and a successful message is displayed: "Update successful"
+
+2) Mark as "closed"
+
+After clicking on it, the status of the selected messages are changed for "closed" and a successful message is displayed: "Update successful"
+
+3) Mark as "pending 1"
+
+After clicking on it, the status of the selected messages are changed for "pending 1" and a successful message is displayed: "Update successful"
+
+4) Mark as "pending 2"
+
+After clicking on it, the status of the selected messages are changed for "pending 2" and a successful message is displayed: "Update successful"
+
+5) Delete selection
+
+After clicking on it, a modal is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the selected messages are deleted, a successful message is displayed "The selection has been successfully deleted."
+
+### Listing
+
+**The messages sent through the contact form in the front office are only listed here if the contact ('Webmaster' or 'Customer service', for example) has the "Save messages?" option enabled.**
+
+This setting can be changed in Shop parameters > Contact > Contact, when editing the contacts.
+
+If the option is disabled for the contact that the customer chooses, the message is simply sent to the contact's email address, and is not stored in PrestaShop.
+
+#### Multistore
+
+Only the messages created in the selected store or store group (through the context) are displayed.
+
+For example, let's say I have the following configuration:
+- Group 1
+  - Shop A
+  - Shop B
+- Group 2
+  - Shop C
+  
+If I select group 1 as context, then the messages created in Shop A and Shop B will be displayed.
+
+If I select Shop A as context, then the messages created in Shop A will be displayed.
+
+If I select all shops as context, then the messages created in Shop A, Shop B and Shop C will be displayed.
+
+#### Select all / unselect all checkbox
+
+After checking the box, all the messages displayed on the page are selected or unselected.
+
+#### Columns
+
+There are 12 columns:
+
+1) ID
+
+It's possible to sort by id asc or desc and search by id.
+
+2) Customer
+
+The firstname and lastname are displayed if the customer was logged in while sending the message. Otherwise, "--" is displayed.
+
+When it is the merchant, customer service, or any other contact, who writes to a customer, the customer's first name and last name are displayed too.
+
+It's possible to sort by customer asc or desc and search by customer (firstname and/or lastname)
+
+3) Email
+
+The customer's email address is displayed.
+It's possible to sort by email asc or desc and search by email
+
+4) Type (contact)
+
```

Comment:
I don't know if it's useful to add this information, but when the contact is made via the "Add a message" in FO > User Account > Order History > Order, there is currently no "Type" of Contact in the listing (so "--" is displayed).

Reply:
Yes, good idea :+1: 
I added this here: https://github.com/PrestaShop/prestashop-specs/pull/187/commits/1c3f188a68e773b4f34de83849cce17b4e3c725b